### PR TITLE
Bench code simplification (PR#1 of 2)

### PR DESCRIPTION
**Pull request by ibgreen**
Prep diff for benchmarking of async methods.

### CODE REVIEW

File: modules/bench/src/bench-loggers.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+      break;
+    case LOG_ENTRY.COMPLETE:
+      console.log('');
+      console.log(`Completed benchmark in ${time}s`);
+      break;
+    default:
+  }
+}
+
+export function logResultsAsTreeWithElapsed({entry, id, itersPerSecond, error, time}) {
```
Comment (Pessimistress) [inline modules/bench/src/bench-loggers.js]: Just add a `elapsed` flag to `logResultsAsTree` and get rid of this function?

File: modules/bench/src/bench.js [hunk 2]
```diff
@@ -138,56 +146,61 @@ function runCalibrationTests({tests}) {
...
+function logEntry(test, opts) {
+  const priority = (global.probe && global.probe.priority) | 10;
+  if ((opts.priority | 0) <= priority) {
+    test.opts.log(opts);
+  }
+}
+
 // Run a list of bench test case async
-function runAsyncTests({tests, onBenchmarkComplete = noop}) {
+function runTestsAsynchronously({tests, onBenchmarkComplete = noop}) {
```
Comment (Pessimistress) [inline modules/bench/src/bench.js]: Nit: usually these would be named `runTests` and `runTestsSync`

File: modules/bench/src/bench-loggers.js (hunk 1 above)
Comment (ibgreen) [reply modules/bench/src/bench-loggers.js]: > Just add a `elapsed` flag to `logResultsAsTree` and get rid of this function?

These are functions that are intended to be supplied as callbacks to `Bench`, so the parameters cannot be controlled at call time, unless we e.g. add options forwarding.

File: modules/bench/src/bench.js (hunk 2 above)
Comment (ibgreen) [reply modules/bench/src/bench.js]: > Nit: usually these would be named `runTests` and `runTestsSync`

OK dropped `Asynchronously`. 

This was not intended to offer such an interface distinction, more an attempt to create a more self-documenting name for the function, as the old name was rather confusing (at least in context).

**Pull request merged**